assets/xml/objects/object_tru.xml

```diff
@@ -1,108 +1,117 @@
+        <DList Name="gKoumeTargetDL" Offset="0x390" />
+        <DList Name="gKoumeChainDL" Offset="0x4C8" />
+        <Texture Name="gKoumeChainTex" OutName="koume_chain" Format="rgba16" Width="16" Height="32" Offset="0x560" />
+        <Texture Name="gKoumeTargetSideTex" OutName="koume_target_side" Format="rgba16" Width="32" Height="32" Offset="0x960" />
+        <Texture Name="gKoumeTargetFaceTex" OutName="koume_target_face" Format="rgba16" Width="32" Height="32" Offset="0x1160" />
+
+        <DList Name="gKoumeBottleDL" Offset="0x1F90" />
+        <DList Name="gKoumePotionDL" Offset="0x20C8" />
+        <DList Name="gKoumeEmptyDL" Offset="0x2160" />
+        <Texture Name="gKoumeBottleTex" OutName="koume_bottle" Format="i4" Width="16" Height="16" Offset="0x2168" />
+
+        <Animation Name="gKoumeFlyAnim" Offset="0x2BD8" />
+        <Animation Name="gKoumeTakeOffAnim" Offset="0x3698" />
+        <Animation Name="gKoumeHoverAnim" Offset="0x446C" />
+        <Animation Name="gKoumeDrinkAnim" Offset="0x7FA0" />
+        <Animation Name="gKoumeInjuredRaiseHeadAnim" Offset="0x9348" />
+
+        <DList Name="gKoumePelvisDL" Offset="0xB730" />
+        <DList Name="gKoumeTorsoDL" Offset="0xB940" />
+        <DList Name="gKoumeNeckDL" Offset="0xBA88" />
+        <DList Name="gKoumeHeadDL" Offset="0xBBB8" />
+        <DList Name="gKoumeLeftBraidStartDL" Offset="0xC630" />
+        <DList Name="gKoumeLeftBraidEndDL" Offset="0xC730" />
+        <DList Name="gKoumeTeethDL" Offset="0xC810" />
+        <DList Name="gKoumeRightBraidStartDL" Offset="0xC860" />
+        <DList Name="gKoumeRightBraidEndDL" Offset="0xC960" />
+        <DList Name="gKoumeRightSleeveStartDL" Offset="0xCA40" />
+        <DList Name="gKoumeRightSleeveFrontDL" Offset="0xCB50" />
+        <DList Name="gKoumeRightHandDL" Offset="0xCCC0" />
+        <DList Name="gKoumeRightSleeveMidDL" Offset="0xCFA8" />
+        <DList Name="gKoumeRightSleeveEndDL" Offset="0xD0C0" />
+        <DList Name="gKoumeLeftSleeveStartDL" Offset="0xD1D0" />
+        <DList Name="gKoumeLeftSleeveFrontDL" Offset="0xD2E0" />
+        <DList Name="gKoumeLeftHandDL" Offset="0xD450" />
+        <DList Name="gKoumeBroomDL" Offset="0xD738" />
+        <DList Name="gKoumeLeftSleeveMidDL" Offset="0xDA38" />
+        <DList Name="gKoumeLeftSleeveEndDL" Offset="0xDB50" />
+        <DList Name="gKoumeRightThighDL" Offset="0xDC40" /> <!-- Renders a single triangle instead of a normal-looking limb -->
+        <DList Name="gKoumeRightShinDL" Offset="0xDC88" />
+        <DList Name="gKoumeRightFootDL" Offset="0xDD38" />
+        <DList Name="gKoumeLeftThighDL" Offset="0xDDD0" /> <!-- Renders a single triangle instead of a normal-looking limb -->
+        <DList Name="gKoumeLeftShinDL" Offset="0xDE18" />
+        <DList Name="gKoumeLeftFootDL" Offset="0xDEC8" />
+
+        <Animation Name="gKoumeInjuredTalkingAnim" Offset="0xEEDC" />
+        <Animation Name="gKoumeInjuredLyingDownAnim" Offset="0xF9A0" />
```

Comment:
Slightly different verb tense than the other animations. Maybe `gKoumeInjuredTalkAnim` and `gKoumeInjuredLieDownAnim`?

Reply:
I agree with the `talk` but I'd prefer `InjuredLyingDown`. `InjuredLieDown` seems more like the animation is of her lying down (i.e. the process to lie down), rather than its the animation when she is lying down injured.

Reply:
Yeah, that's fine. I think I did something like that in `object_ma1` too for that exact reason.